### ISSUE TITLE
axum_debug reports error when a handler has multiple extractors.

### ISSUE THREAD

**Issue by kenmasumitsu** [tokio-rs/axum#551]
### Version

axum-debug = "0.2.1"
[69fee5864d9463d3646e89cf055019bc702c297b ](https://github.com/tokio-rs/axum/commit/69fee5864d9463d3646e89cf055019bc702c297b)

### Platform

Darwin 20.6.0 Darwin Kernel Version 20.6.0: Tue Oct 12 18:33:42 PDT 2021; root:xnu-7195.141.8~1/RELEASE_X86_64 x86_64

### Description

<short summary of the bug>

I tried this code:

```

#[debug_handler]
async fn get_user_things(
    Path(user_id): Path<Uuid>,
    pagination: Option<Query<Pagination>>,
) -> impl IntoResponse {
    let Query(pagination) = pagination.unwrap_or_default();

    println!("{:?}, {:?}", user_id, pagination);
    Html("<h1>Hello, World!</h1>")
}

```

I expected to see this happen: No error reports.

Instead, this happened: shows the report below

```
the name `__axum_debug_check_get_user_things_from_request` is defined multiple times
`__axum_debug_check_get_user_things_from_request` must be defined only once in the value namespace of this modulerustcE0428
main.rs(48, 20): previous definition of the value `__axum_debug_check_get_user_things_from_request` here
```
-->

**Comment by davidpdrsn** [comment]
I know exactly that the issue is. I'll get it fixed 😊

**Comment by davidpdrsn** [comment]
Are you able to test https://github.com/tokio-rs/axum/pull/552?

**Comment by kenmasumitsu** [comment]
I confirmed the issue was resolved with #552.

**Comment by davidpdrsn** [comment]
Fix is shipped in axum-debug v0.2.2

**Comment by kenmasumitsu** [comment]
I appreciate your quick fix!
